ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/dynamic-prefix-lists.j2

```diff
@@ -0,0 +1,25 @@
+{% if dynamic_prefix_lists is arista.avd.defined %}
+
+## Dynamic Prefix-lists
+
+### Dynamic Prefix-lists Summary
+
+{%     for dynamic_prefix_list in dynamic_prefix_lists | arista.avd.natural_sort('name') %}
+#### {{ dynamic_prefix_list.name }}
+
+{%         if dynamic_prefix_list.prefix_list.ipv4 is arista.avd.defined %}
+{%             set prefix_list = dynamic_prefix_list.prefix_list.ipv4 %}
+{%         elif dynamic_prefix_list.prefix_list.ipv6 is arista.avd.defined %}
+{%             set prefix_list = dynamic_prefix_list.prefix_list.ipv6 %}
+{%         endif  %}
```

Comment:
line 10-14 can be simplified to ```set prefix_list = dynamic_prefix_list.prefix_list.ipv4 | arista.avd.default(dynamic_prefix_list.prefix_list.ipv6, '-')```

Reply:
Great point, you mentioned this method in another review, and I did not copy the approach here.